src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -148,22 +162,26 @@
     margin-top: 30px;
   }
 
+  .paragraph3Wrapper {
+    margin-top: 30px;
+  }
+
   .ctaTitleBlock {
     margin-top: 40px;
     max-width: 100%;
   }
 
-  .image1Wrapper {
+  .leftTopImageWrapper {
     display: none;
   }
 
-  .image2Wrapper {
+  .leftBottomImageWrapper {
     display: none;
   }
 
-  .image3Wrapper {
+  .rightImageWrapper {
+    margin-right: -40px;
     margin-top: 40px;
-    overflow: hidden;
```

Comment:
I think if we take this out, it causes the whole page to scroll if this thing expands off screen. That said, I think we actually need to go through and just set `overflow-x: hidden` on all the bleed wrappers, since that would allow us to enforce this for all content under each bleed wrapper, rather than individually doing this for each thing that may possible spill off screen.

Reply:
I added `overflow-x: hidden` to the bleed wrapper. Do I still need to set `overflow: hidden;` to `.leftBottomImageWrapper.solo` if I have `.image` set to `max-width: 100%;`?

Reply:
Let's not worry about that for now. Technically, it could overflow if the minimum width is greater than 100%, and also possibly in a few other cases, but it's unlikely to happen in our situation.

Reply:
Okay, I'll keep the `overflow` property. Thanks!